include/wlr/types/wlr_drm_lease_v1.h

```diff
@@ -0,0 +1,141 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_DRM_LEASE_V1_H
+#define WLR_TYPES_WLR_DRM_LEASE_V1_H
+
+#include <wayland-server.h>
+#include <wlr/backend.h>
+
+struct wlr_drm_backend;
+
+struct wlr_drm_lease_manager_v1 {
+	struct wl_list resources; // wl_resource_get_link
+	struct wl_global *global;
+	struct wlr_drm_backend *backend;
+
+	struct wl_list connectors; // wlr_drm_lease_connector_v1::link
+	struct wl_list leases; // wl_resource_get_link
+	struct wl_list lease_requests; // wl_resource_get_link
+
+	struct {
+		/**
+		 * Upon receiving this signal, call
+		 * wlr_drm_lease_manager_v1_grant_lease_request to grant a lease of the
+		 * requested DRM resources, or
+		 * wlr_drm_lease_manager_v1_reject_lease_request to reject the request.
+		 */
+		struct wl_signal lease_requested; // wlr_drm_lease_request_v1
+		/**
+		 * Upon receiving this signal, it is safe to re-use the leased
+		 * resources. After the signal is processed, the backend will re-signal
+		 * new_output events for each leased output.
+		 *
+		 * The lifetime of the lease reference after the signal handler returns
+		 * is not defined.
+		 */
+		struct wl_signal lease_destroyed; // wlr_drm_lease_v1
```

Comment:
This should be an event in `struct wlr_drm_lease_v1`

Reply:
I haven't forgotten this, either.